managed/src/main/java/com/yugabyte/yw/commissioner/HealthChecker.java

```diff
@@ -211,7 +211,7 @@ private void processResults(Customer c, Universe u, String response, long durati
       SmtpData smtpData = emailHelper.getSmtpData(c.uuid);
       if (!StringUtils.isEmpty(emailDestinations) && (smtpData != null)
           && (sendMailAlways || hasErrors)) {
-        String subject = String.format("%s - <%s> %s", hasErrors ? "ERROR" : "OK", c.getTag(),
+        String subject = String.format("%s %s - <%s> %s", "Yugabyte Platform Alert " , hasErrors ? "ERROR" : "OK", c.getTag(),
```

Comment:
- Why "Yugabyte Platform Alert" as a parameter?
- Remove space at the end of "Yugabyte Platform Alert ".
- But overall: I disagree with the new subject. As example, my previous headers for successful health-checks were "OK - <[admin][cloud]> spotachev-test",   in the new format they will be "Yugabyte Platform Alert OK - <[admin][cloud]> spotachev-test". I don't think we need "Yugabyte Platform Alert" here for successful health-checks.

Reply:
The intention is to keep the same email alert format for all the email alert

THought to just append in each email subject **Yugabyte Platform Alert**
Yugabyte Platform Alert - <[admin][admin]>
ERROR - <[admin][admin]> jitu-test-gcp-tls
OK - <[admin][admin]> tls-disabled-aws